README.md

```diff
@@ -62,11 +62,32 @@ There is only one modifiable property in settings:
    //    - "on"
    //    - "off"
    //    - "smart": Automatic formatting is only enabled if there is a `black` section in the project's `pyproject.toml`
-   "format_on_save": "on"
+   "format_on_save": "on",
+   // Black [OPTIONS]
+   "line_length": "88",
```

Comment:
The type of `line_length` should be `int`.

Reply:
Sure, I'll fix it! Thanks.